pulp_docker/app/serializers.py

```diff
@@ -14,27 +14,6 @@
 from . import models
 
 
-class ManifestListTagSerializer(SingleArtifactContentSerializer):
-    """
-    Serializer for ManifestListTags.
-    """
-
-    name = serializers.CharField(help_text="Tag name")
-    manifest_list = DetailRelatedField(
-        many=False,
-        help_text="Manifest List that is tagged",
-        view_name='docker-manifest-lists-detail',
-        queryset=models.ManifestList.objects.all()
-    )
-
-    class Meta:
-        fields = SingleArtifactContentSerializer.Meta.fields + (
-            'name',
-            'manifest_list',
-        )
-        model = models.ManifestListTag
-
-
 class ManifestTagSerializer(SingleArtifactContentSerializer):
```

Comment:
you forgot to change the name s/manifest/tagged_manifest
https://github.com/pulp/pulp_docker/blob/9f3a81ed2456acaf2a108d33c685657f60231148/pulp_docker/app/serializers.py#L23
https://github.com/pulp/pulp_docker/blob/9f3a81ed2456acaf2a108d33c685657f60231148/pulp_docker/app/serializers.py#L33